exporters/committime/collector_base.py

```diff
@@ -191,7 +195,12 @@ def get_metric_from_build(self, build, app, namespace, repo_url):
 
             metric.commit_hash = commit_sha
             metric.name = app
-            metric.committer = build.spec.revision.git.author.name
+
+            if build.spec.revision is None:
```

Comment:
Same as above, get it when metric.committer is not valid from the:
`
metric.committer = self._get_author_from_build(build)
`

I don't know if such use case is valid when information is missing from build.spec.revision and have generic option to fill that data in the build?